plugins/modules/redfish_storage_volume.py

```diff
@@ -656,6 +787,67 @@ def validate_inputs(module):
                          " volume_id must be specified to perform further actions.")
 
 
+def perform_force_reboot(module, session_obj):
+    payload = {"ResetType": "ForceRestart"}
+    job_resp_status, reset_status, reset_fail = wait_for_redfish_reboot_job(session_obj, SYSTEM_ID, payload=payload)
+    if reset_status and job_resp_status:
+        job_uri = MANAGER_JOB_ID_URI.format(job_resp_status["Id"])
+        resp, msg = wait_for_job_completion(session_obj, job_uri, wait_timeout=300)
+        if resp:
+            job_data = strip_substr_dict(resp.json_data)
+            if job_data["JobState"] == "Failed":
+                module.exit_json(msg=REBOOT_FAIL, failed=True)
+        else:
+            module.exit_json(msg=msg)
```

Comment:
This is a timeout message i believe, good to return job details in `job_status` as well.